backend/src/StamAcasa.JobScheduler/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using AutoMapper;
+using EasyNetQ;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Hosting;
+using StamAcasa.Common;
+using StamAcasa.Common.Notifications;
+using StamAcasa.Common.Queue;
+using StamAcasa.Common.Services;
+using StamAcasa.JobScheduler.Extensions;
+using StamAcasa.JobScheduler.Jobs;
+
+namespace StamAcasa.JobScheduler
+{
+    internal static class Program
+    {
+        private static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        private static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureServices((hostContext, services) =>
+                {
+                    services.AddHostedService<JobScheduler>()
+                        .AddLogging();
+
+                    services.AddAutoMapper(typeof(Program), typeof(UserDbContext));
+                    services.AddDbContext<UserDbContext>(options =>
+                        options.UseSqlite(hostContext.Configuration.GetConnectionString("UserDBConnection")));
+
+                    var hostName = hostContext.Configuration["RabbitMQ:HostName"];
+                    var userName = hostContext.Configuration["RabbitMQ:User"];
+                    var port = ushort.Parse(hostContext.Configuration["RabbitMQ:Port"]);
+                    var password = hostContext.Configuration["RabbitMQ:Password"];
+
+                    services.TryAddSingleton(sp => RabbitHutch.CreateBus(
```

Comment:
My only suggestion here is to use the newly developed `IQueueService` in Common.
It would only make it easier to interface with the queue.

Reply:
It should already be using `IQueueService` (have a look at line 53). The `IBus` returned by that factory method is just a dependency of the `QueueService` implementation.

Reply:
Yes, you are right, I missed that.